src/main/java/co/uk/rushorm/core/RushSearch.java

```diff
@@ -312,7 +312,9 @@ public RushSearch whereIN(String column, List<String> values) {
         for (String value : values) {
             inStatement += value + ",";
         }
-        inStatement = inStatement.substring(0, inStatement.length() - 1) + ")";
+        if(!values.isEmpty())
```

Comment:
Happy with this check but please add { } after the if to match project style. Thanks